test_infra/scripts/deploy-base.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+set -e
+
+pushd ..
+cdk bootstrap
```

Comment:
Hmm I am not sure we should be responsible for the bootstrapping of the account? User might have already done it or is using a custom bootstrap template. Perhaps clearly explaining in the contributing guide that the account should be bootstrapped is enough?

Reply:
The reasoning behind it was that I'd like to simplify the instructions as much as possible. Less instructions to follow - users are less likely to get scared by them. If the account is bootstrapped already, there is no harm in running the command again. 

One point to ponder though: there may be issues if it's bootstrapped with a different version - I have seen failures due to that. Then having this script may obscure a bit what exactly is going on, although, the cdk stdout is hard to ignore :) and I think it will tell you the version difference.

Reply:
Fair enough, we can keep the bootstrap part